### PR TITLE
Blog Post

### PR DESCRIPTION
**Pull request by kathweaver**
Blog post on contributing

Let me know if this needs to be changed in anyway.

### CODE REVIEW

File: _posts/blog/kweaver01_18_2016.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+###Contributing to Open Live Writer
+I have been using the original Live Writer since it came out, and loved it.  I was 
+am avid user of the plug-ins too.
```
Comment (ScottIsAFool) [inline _posts/blog/kweaver01_18_2016.md]: Typo: "am"

**Comment by kathweaver** [comment]
Thanks, fixed.

**Comment by kathweaver** [comment]
Thanks so now how do I get to it?

**Comment by ScottIsAFool** [comment]
Not a scooby :)

**Pull request merged**